src/components/Form/Form.test.tsx

```diff
@@ -0,0 +1,101 @@
+import React from "react";
+import "@testing-library/jest-dom/extend-expect";
+import {
+  Form,
+  Input,
+  Button,
+  useBasisForm,
+} from "../..";
+
+import { render, screen, userEvent, act, fireEvent } from "../../utils/test";
+import { SubmitHandler } from "react-hook-form";
+
+
+interface SimpleFormValues {
+  testInput: string;
+}
+
+interface SimpleProps {
+  onSubmit: SubmitHandler<SimpleFormValues>;
+  validate?: any;
+}
+
+const SimpleForm = ({ onSubmit, validate }: SimpleProps) => {
+  const { methods, Field } = useBasisForm<SimpleFormValues>();
+
+  return (
+    <Form methods={methods} onSubmit={onSubmit}>
+      <Field label="Test" name="testInput" as={Input} validate={validate} />
+      <Button type="submit">Submit</Button>
+    </Form>
+  )
+}
+
+describe('Form', () => {
+  it('should render the form', () => {
+    render(<SimpleForm onSubmit={() => { }} />)
+
+    const input = screen.getByLabelText("Test");
+    expect(input).toBeInTheDocument();
+  });
+
+  it('should display required error when input blurred without value', async () => {
+    render(<SimpleForm onSubmit={() => { }} />)
+
+    const input = screen.getByLabelText("Test");
+
+    await act(async () => {
+      await userEvent.tab();
```

Comment:
Add a comment here:
> Focus the input

and on the next line:
> Blur the input

so it's clear why we do tab twice.